Lil-Peep/src/styles/settings/variables.scss.liquid

```diff
@@ -3,12 +3,23 @@ $color-primary: {{ settings.color_primary }};
 
 // Text colors
 $color-body-text: {{ settings.color_body_text }};
+$color-categories-text: #627387;
+$color-white-font: #FFFFFF;
+$color-text-hover: #FF0000;
+$color-theme-lilpeep: #72808C;
 
 // Backgrounds
 $color-body: #fff;
+$color-white: #ffffff;
```

Comment:
@hrodriguez1 Use only three characters when using hexadecimal values with the same letter.
This should be #fff instead of #ffffff